easybuild/easyblocks/s/suitesparse.py

```diff
@@ -122,7 +124,7 @@ def configure_step(self):
         if cfgvars:
             try:
                 f = open(fp, "a")
-                f.write("# lines below added automatically by EasyBuild")
+                f.write("# lines below added automatically by EasyBuild\n")
```

Comment:
Yes. I saw it when I removed the LAPACK and BLAS variables, and thanks to the sanity check on libmetis.so not being built.